src/pages/coursesPage.jsx

```diff
@@ -1,8 +1,29 @@
+import CourseCard from "../components/courseCard";
+import PageHeader from "../components/pageHeader";
+import { clubs } from "../constants/golfClubs";
+
 const CoursesPage = () => {
   return (
     <>
-      <div>
-        <h1>Courses</h1>
+      <PageHeader title="Courses" />
+      <hr className="border-black" />
+      <div className="bg-[#D9D9D9]">
+        <div className="max-w-5xl mx-auto py-8 px-4 sm:px-6 lg:px-8">
+          <div className="">
+            {clubs
+              .sort((a, b) => +(a.name > b.name) * 2 - 0.5)
```

Comment:
name this function maybe ?
```suggestion
              .sort((a, b) => azCompare(a.name, b.name))
```